str.go

```diff
@@ -13,14 +19,17 @@ import (
 // STR should be final
 type SignedTreeRoot struct {
 	tree        *MerkleTree
-	counter     int64
-	prevCounter int64
+	counter     uint64
```

Comment:
If the ``epoch`` is a monotonically increasing integer, then ``uint64`` seems like a perfectly valid type. My issue was more that ``counter`` isn't a very descriptive name for what it actually represents, the ``epoch``. The CONIKS design doesn't require that ``epoch`` be of a specific type, it only represents a point in the history of the PAD at which we take a snapshot.